walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,65 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+
 
 public final class FindMeetingQuery {
+
+  public Collection<TimeRange> mergeTimes(Collection<Event> events, MeetingRequest request) {
```

Comment:
Can this method be private? Can you also include a comment describing what mergeTimes does?

Reply:
It can, will fix and add a comment!